rpcs3/util/dyn_lib.hpp

```diff
@@ -60,14 +60,20 @@ namespace utils
 	// (assume the lib is always loaded)
 	void* get_proc_address(const char* lib, const char* name);
 
-	template <typename F>
+	template <typename CallingConventionTag, typename F>
```

Comment:
All the changes in this file are neither explained nor have any relation to the rest of the code